inception-project/inception
# #1765 Curation Sidebar: Should be able to change user even when finished

**What's in the PR**
- [x] always allow to change settings
- [x] update message that doc is finished to include username
- [x] fix user in annotator state not being up to date with curator
- [x] should also be able to show my annotation user's annotations during curation with CURATION_USER
- [x] fix that CURATION_USER document can be finished but this does not update to document state (as seen on monitoring page)

**How to test manually**
Checkout webanno branch for https://github.com/webanno/webanno/issues/1822
* Enable the curation sidebar with `curation.sidebar.enabled=true` in your settings.properties
* Be an annotator and curator
* Annotate a document as yourself and finish it
* You should be able to switch to the CURATION_USER via the curation sidebar to curate it

**Automatic testing**

**Documentation**
